### DISCUSSION TITLE
Introduction and Doubt

### DISCUSSION THREAD

**Discussion by kirtanchandak** [ruxailab/RUXAILAB#321] [General]
Hello Everyone,

I'm Kirtan, an undergraduate student majoring in Computer Science from India, graduating in 2025. I have prior experience working with a web3 company as a full-stack developer, and I am proficient in writing production-grade frontend and backend applications.

I am eager to contribute my knowledge to ruxailab. I have set up the code for https://github.com/ruxailab/web-eye-tracker-front, and I have a question regarding where I should start contributing within the organization. Should I continue working on this repository, or should I begin exploring the main repository? (@marcgc21)

Thank You.

**Comment by jvJUCA** [comment]
Dear @kirtanchandak  ,

Sorry for the delay in responding, your willingness to contribute is highly appreciated. You are more than welcome to contribute to any repository within our organization. Your input and efforts are valuable to us, and we encourage you to engage with any projects you find interesting or feel you can assist with.

Thank you for being open to collaboration and helping us make progress together.

Best regards,

@jvJUCA
